coral/types/utils.d.ts

```diff
@@ -16,10 +16,17 @@ type ResolveIntersectionTypes<T> = {
   // eslint-disable-next-line @typescript-eslint/ban-types
 } & {};
 
+type Paginated<T> = {
```

Comment:
Very nice to have move this type here ✨  Could we maybe take the opportunity to update the usage of in in `topic-types` and `acl-types`? ^^

Reply:
Sure! 

I was a bit cautious to not interfere with open work too much but will add it here.

Reply:
Updated!

Reply:
I think that kind of `domain` level changes are ok. This will interfer way more haha! https://github.com/aiven/klaw/pull/646

In a good way xd